auth-web/tests/unit/components/DeactivateCard.spec.ts

```diff
@@ -21,55 +24,73 @@ describe('Deactivated card.vue', () => {
   let wrapper: any
   const localVue = createLocalVue()
   localVue.use(Vuex)
+  localVue.use(i18n)
 
   it('Truthy and basic test', () => {
     wrapper = mount(DeactivateCard, {
       vuetify,
       localVue,
       router,
       mocks: {
-        $t: (mock) => mock
+        t: (mock) => mock
```

Comment:
for this and all the other tests: you've added a mock for i18n already so you don't need to mock out t. It's a better test if t is not mocked otherwise you can commit bugs like before with the test passing